libmscore/line.cpp

```diff
@@ -679,6 +679,7 @@ QPointF SLine::linePos(Grip grip, System** sys) const
                                     BarLineType blt = toBarLine(e)->barLineType();
                                     switch (blt) {
                                           case BarLineType::END_REPEAT:
+                                          case BarLineType::END_START_REPEAT:
```

Comment:
This shouldn't be needed either if the barline type never appears in the score.

Reply:
except to avoid a compuler warning about an unhandled case